tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -2063,14 +2067,16 @@ std::unique_ptr<Thunk> IrEmitterUnnested::BuildWhileThunk(
 
   // Generate thunk sequence for while 'condition'.
   HloComputation* condition = hlo->while_condition();
-  IrEmitterUnnested ir_emitter_condition(hlo_module_config_, condition,
-                                         ir_emitter_context_);
+  IrEmitterUnnested ir_emitter_condition(
+      hlo_module_config_, condition, ir_emitter_context_,
```

Comment:
Could we make TargetIRBuilder (or TargetMachineFeatures) a property of ir_emitter_context_?  That's kind of the purpose of the class anyway.

Reply:
(Although see below, I think we may be able to get rid of TargetIRBuilder altogether.)